grapher/timeline/TimelineController.ts

```diff
@@ -165,15 +167,17 @@ export class TimelineController {
         this.updateEndTime(endTime)
     }
 
+    // TODO
```

Comment:
Still needed, or Much Todo About Nothing?

Reply:
Haha I really wanted to emphasise it and I forgot it anyway in the end 🤦